MusicHub/tracks/migrations/0003_alter_track_track_length.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2022-09-23 11:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tracks', '0002_alter_track_track'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='track',
+            name='track_length',
+            field=models.PositiveIntegerField(blank=True, null=True),
+        ),
+    ]
```

Comment:
there are 3 migrations for the same app per one PR. Try to squash migrations for next PRs